microprofile-jwt-auth-tool/src/main/java/org/jboss/eap/qe/microprofile/jwt/auth/tool/JwtHelper.java

```diff
@@ -0,0 +1,69 @@
+package org.jboss.eap.qe.microprofile.jwt.auth.tool;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.Signature;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.UUID;
+
+/**
+ * Sugar syntax for generating JWTs with re-used values
+ */
+public final class JwtHelper {
+    private final RsaKeyTool keyTool;
+    private final String issuer;
+
+    public JwtHelper(final RsaKeyTool keyTool, final String issuer) {
+        this.keyTool = keyTool;
+        this.issuer = issuer;
+    }
+
+    /**
+     * Generates a spec compliant base64-encoded signed JWT that expires after one hour and has the claims "sub" and
+     * "preferred_username" set to "FAKE_USER".
+     *
+     * @return a base64-encoded signed JWT token.
+     */
+    public JsonWebToken generateProperSignedJwt() {
```

Comment:
FAKE_USER should be reflected in method name.

Reply:
I think that the method definition with no arguments implies that all "claim" values of the JWT are mocked.